android/src/main/java/com/swmansion/reanimated/layoutReanimation/ScreensHelper.java

```diff
@@ -0,0 +1,66 @@
+package com.swmansion.reanimated.layoutReanimation;
+
+import android.view.View;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+
+public class ScreensHelper {
+
+  public static View getTabNavigator(View view) {
+    View currentView = view;
+    while (currentView != null) {
+      if (currentView.getClass().getSimpleName().equals("ScreenContainer")) {
```

Comment:
I already saw this check in `ReanimatedNativeHierarchyManager.java`, maybe we can extract it into a separate static method like `isViewScreenContainer`? Same for `Screen` and `ScreenCoordinatorLayout`.